OpenRA.Renderer.Sdl2/Sdl2GraphicsDevice.cs

```diff
@@ -246,26 +246,11 @@ public void SetBlendMode(BlendMode mode)
 					ErrorHandler.CheckGlError();
 					GL.BlendFunc(BlendingFactorSrc.OneMinusDstColor, BlendingFactorDest.One);
 					break;
-				case BlendMode.Translucency25:
+				case BlendMode.Translucency:
```

Comment:
Having a dedicated Translucency blend mode but then allowing any other mode to set its own Alpha (translucency) value smells a bit like spaghetti.